packages/gatsby-source-medium/src/gatsby-node.js

```diff
@@ -8,6 +8,19 @@ const fetch = username => {
 
 const prefix = `])}while(1);</x>`
 
+const convertTimestamps = (nextObj, prevObj, prevKey) => {
+  if (typeof nextObj === 'object') {
+    Object.keys(nextObj).map(key => convertTimestamps(nextObj[key], nextObj, key));
+  } else {
+    if (typeof nextObj === 'number' && nextObj >> 0 !== nextObj) {
```

Comment:
Is there a lodash helper method you can use here? I have no idea what you're doing 😅

Also… is the Medium schema not set? Don't we just know what the schema will be?

Reply:
Maybe there's a lodash helper method for this but I'm not sure. 

It's recursively crawling down all nested objects until it finds a primitive value then it checks if it's a number and if that number overflows a 32-bit integer.

This method would be schema agnostic which wouldn't break if it were to change also I'm not sure if there is much advantage to tie it to the Medium as this could be used elsewhere where the source provides timestamps instead of ISO 8601 date string (which Gatsby uses)

Reply:
Will date fields *always* be a number which overflows a 32-bit integer? We're not going to get into a situation where some date fields are numbers and others are strings?

Reply:
For Medium this shouldn't be a problem because we are not going to see posts from the 70's even then it would only miss seven weeks from 7 December 1969 to 25 January 1970

Reply:
Cool, let's get this published! Thanks for the PR!